src/test/java/org/jabref/logic/importer/fetcher/DOABFetcherTest.java

```diff
@@ -0,0 +1,244 @@
+        Antonios_Tsourdos.setField(StandardField.URI, "https://directory.doabooks.org/handle/20.500.12854/61443");
+        Antonios_Tsourdos.setField(StandardField.ABSTRACT, "ctive technological development has fuelled " +
+                "rapid growth in the number of Unmanned Aerial Vehicle (UAV) platforms being deployed around" +
+                " the globe. Novel UAV platforms, UAV-based sensors, robotic sensing and imaging techniques," +
+                " the development of processing workflows, as well as the capacity of ultra-high temporal and" +
+                " spatial resolution data, provide both opportunities and challenges that will allow engineers" +
+                " and scientists to address novel and important scientific questions in UAV and sensor design," +
+                " remote sensing and environmental monitoring. This work features papers on UAV sensor design," +
+                " improvements in UAV sensor technology, obstacle detection, methods for measuring optical " +
+                "flow, target tracking, gimbal influence on the stability of UAV images, augmented reality " +
+                "tools, segmentation in digital surface models for 3D reconstruction, detection, location and " +
+                "grasping objects, multi-target localization, vision-based tracking in cooperative multi-UAV " +
+                "systems, noise suppression techniques , rectification for oblique images, two-UAV " +
+                "communication system, fuzzy-based hybrid control algorithms, pedestrian detection and tracking" +
+                " as well as a range of atmospheric, geological, , agricultural, ecological, reef, wildlife, " +
+                "building and construction, coastal area coverage, search and rescue (SAR), water plume " +
+                "temperature measurements, aeromagnetic and archaeological surveys applications");
+        Antonios_Tsourdos.setField(StandardField.LANGUAGE, "English");
+        Antonios_Tsourdos.setField(StandardField.KEYWORDS, "UAV-Based Remote Sensing, drones, pedestrian detection and tracking as well as a range of atmospheric, UAV images, fuzzy-based hybrid control algorithms, water plume temperature measurements, aeromagnetic and archaeological surveys, geological, two-UAV communication system, agricultural, wildlife, reef, detection, multi-target localization, ecological, search and rescue (SAR), coastal area coverage, remote sensing, environmental science, segmentation in digital surface models for 3D reconstruction, location and grasping objects, augmented reality tools, building and construction, aerial robotics, noise suppression techniques, rectification for oblique images, vision-based tracking in cooperative multi-UAV systems");
+        Antonios_Tsourdos.setField(StandardField.PUBLISHER, "MDPI - Multidisciplinary Digital Publishing Institute");
+    }
+
+    @Test
+    public void TestGetName() {
```

Comment:
Method names should start with a lower case:

```suggestion
    public void testGetName() {
```